WooCommerce/Classes/ViewRelated/Orders/FulfillViewController.swift

```diff
@@ -330,10 +394,135 @@ extension FulfillViewController: UITableViewDelegate {
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         tableView.deselectRow(at: indexPath, animated: true)
+
+        switch sections[indexPath.section].rows[indexPath.row] {
+        case .trackingAdd:
+            let viewModel = AddTrackingViewModel(siteID: order.siteID, orderID: order.orderID)
+            let addTracking = ManualTrackingViewController(viewModel: viewModel)
+            let navController = WooNavigationController(rootViewController: addTracking)
+            present(navController, animated: true, completion: nil)
```

Comment:
Style nitpick: add a new line before the next case, to give each block of code some vertical breathing room.